client/src/main/java/com/aliyun/emr/rss/client/ShuffleClient.java

```diff
@@ -193,7 +194,7 @@ public abstract RssInputStream readPartition(
       String applicationId, int shuffleId, int partitionId, int attemptNumber) throws IOException;
 
   /**
-   * 注销
+   * unregisteration
```

Comment:
Implement a suitable comment and complete the whole comment(also param) or just delete it.